core/amber/src/main/scala/edu/uci/ics/texera/web/service/ExecutionResultService.scala

```diff
@@ -262,22 +267,12 @@ class ExecutionResultService(
       })
     )
 
-    // first clear all the results
-    sinkOperators.clear()
+//    // first clear all the results
+//    sinkOperators.clear()
```

Comment:
Why is this line commented?

Reply:
fixed. now using `ResultStorage.getOpResultStorage(workflowIdentity).clear()`